src/gui/vsAudio.cpp

```diff
@@ -99,6 +99,18 @@ VsAudio::VsAudio(QObject* parent)
 {
     loadSettings();
 
+    if (isBackendAvailable<AudioInterfaceMode::RTAUDIO>()) {
+        if (!jackIsAvailable()) {
+            m_audioBackendComboModel = {"RtAudio"};
+        }
+    } else {
+        if (!jackIsAvailable()) {
+            // shouldn't happen
+            m_audioBackendComboModel = {};
+        }
+        m_audioBackendComboModel = {"JACK"};
```

Comment:
Should this be in an else branch?

Reply:
Yes, although if it hits that "shouldn't happen" case likely other things would break.